docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,816 @@
+
+What can we do to help you succeed? If you have any ideas or suggestions, you’re always welcome to share your thoughts by opening an issue on our repository.
+
+### How likely is it that you would recommend MUI to a friend or colleague?
+
+This year we observed a decrease in Net Promoter Score (NPS): from 62 in 2020 to 46 in 2021.
+56.4% of you were Promoters this year, down almost 6 percentage points from last year (62.2%).
+We're still in a good place — an NPS score between 30 and 70 is considered great.
+But we definitely have much more to do to achieve an excellent score.
+
+<img src="/static/blog/2021-survey/2.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Detractors: 10.88%, Neutrals: 32.69%, Promoters: 56.43%, Overall NPS: 46.
+Scale: -100 to 0 needs improvements, 0 to 30 is good, 30 to 70 is great, and 70 to 100 is excellent." />
+
+<p class="blog-description">1563 out of 1589 answered.</p>
+
+### What is the main benefit you get from using MUI?
+
+<img src="/static/blog/2021-survey/3.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="This question results in 2019 - Design: 248; Time: 193; DX: 128; Components: 89; Customizability: 53; Documentation: 25; Accessibility: 5; Typescript: 5; Community support: 4; This question results in 2021 - Time: 597; Design: 407; Components: 309; DX: 306; Customizability: 181; Documentation: 57; Community support: 26; Accessibility: 24; Performance: 17" />
+
+<p class="blog-description">1422 out of 1589 answered this question in 2021.</p>
+
+Here's what you liked the most about the product in 2021:
+
+- **Community (2.9x):** While appreciation for the community grew by 290%, the community itself doubled in size over that same time frame.
+  It looks like we're witnessing a network effect: with twice as many users, we got three times more value from the community.
+- **Accessibility (2.2x):** The community has definitely noticed the work we have put in to make our components more accessible.
+  Special thanks to Sebastian Silbermann [(@eps1lon)](https://github.com/eps1lon) for his continued help!
+- **Customizability (1.6x):** Developers seem to be appreciating the new capabilities introduced in v5.
+  That said, we recognize that there is still more work to be done here, given the number of requests we've gotten to better solve this problem.
+- **Components (1.6x):** We'll keep adding new components to the library as long as you continue to appreciate them this much!
+  - Component quality (1.0x): You know that we set the bar very high for the quality of our components.
+    That can be challenge as we continue to grow, because we expect nothing but the best from our team.
+    Thankfully, this result shows that our new members have done an excellent job maintaining our standards.
+- **Time (1.4x):** This is an interesting one, because it's not an issue that we specifically addressed.
```

Comment:
I guess we don't really need the comma here

```suggestion
- **Time (1.4x):** This is an interesting one because it's not an issue that we specifically addressed.
```